CHANGELOG.md

```diff
@@ -39,6 +39,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ### Features
 
+* (x/nft) [#12038](https://github.com/cosmos/cosmos-sdk/issues/12038) Migrate `x/nft` to App Wiring.
```

Comment:
I'm not sure "App Wiring" should be the official name

Reply:
I am not sure what we should can it then. I'll remove that from the changelog from now.